translator/translator_select.go

```diff
@@ -323,7 +325,13 @@ func getSpannerSelectQuery(t *Translator, data *SelectQueryMap) (string, error)
 	if data.ColumnMeta.Star {
 		column = STAR
 		// TODO: Instead of exclude keyword, flattern list of all columns from TableConfigurations.
-		column += excludeInternalColumns
+		if t.UseRowTimestamp && t.UseRowTTL {
```

Comment:
We don't need this if we mark these columns as HIDDEN in Spanner Schema.

Reply:
Yes, will evaluate that in next PR